src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -242,3 +242,10 @@
 (defn tx-type-collectible?
   [tx-type]
   (contains? collectible-tx-set tx-type))
+
+(defn convert-to-gwei
+  [n]
+  (-> n
+      money/wei->gwei
+      (money/with-precision 6)
```

Comment:
Where is this 6 coming from? Should it be defined in a constant?

Reply:
Or I'd suggest to have a precision as an argument